sdk/src/Core/Amazon.Runtime/Internal/Util/StringUtils.cs

```diff
@@ -243,6 +243,14 @@ public static string FromList(List<Enum> values)
         /// <returns>Header value representing the list of T</returns>
         public static string FromValueTypeList<T>(List<T> values)  where T : struct
         {
+            // ToString() on boolean types automatically Pascal Cases. Xml-based protocols
+            // are case sensitive and accept "true" and "false" as the valid set of booleans.
+            // and we should be returning the booleans as is.
+            if ((bool)(values?.Any(x => x.GetType() == typeof(bool))))
+            {
+                return FromList(values?.Select(x => x.ToString().ToLower()));
```

Comment:
```suggestion
                return FromList(values?.Select(x => x.ToString(). ToLowerInvariant()));
```

Reply:
Initially I had the same thought and used `ToLowerInvariant()` but I came across a weird warning, which made the SDK not compile:
```
Error CA1308 In method 'FromValueTypeList', replace the call to 'ToLowerInvariant' with 'ToUpperInvariant'.	
```

But, I could probably just suppress the warning since `ToLowerInvariant()` is technically the right choice here.

Reply:
suppressed the warnings and used `ToLowerInvariant()`